include/maidsafe/routing/types.h

```diff
@@ -85,6 +85,9 @@ using HandleGetReturn =
     boost::expected<boost::variant<std::vector<DestinationAddress>, std::vector<byte>>,
                     maidsafe_error>;
 using HandlePutPostReturn = boost::expected<std::vector<DestinationAddress>, maidsafe_error>;
+using HandlePostReturn =
```

Comment:
I think if we have a different type for Post returns, we should rename `HandlePutPostReturn` to `HandlePutReturn`.